module/tools/SystemConfiguration/data/system/default/etc/sudoers.d/wheel

```diff
@@ -1 +0,0 @@
-%wheel ALL=(ALL) ALL
```

Comment:
Still keep it here, I was just mentioning that it also needs to be in the arch install docs

Reply:
Oh, it is lol